### PR TITLE
Feat: refactor of ui_display_weapons

### PR DESCRIPTION
**Pull request by felipeStouvenel**
# Feats
- Refactored ui_display_weapons
- Updated logic
- Fixed some variables being declared multiple times
- Renamed arguments

### CODE REVIEW

File: scripts/scr_ui_display_weapons/scr_ui_display_weapons.gml [hunk 1]
```diff
@@ -8,372 +8,713 @@ function scr_ui_display_weapons(argument0, argument1, argument2) {
...
-
 	// Flip for offhand
-	if (argument0=2) /*and (argument1=0)*/ and (ui_xmod[rl]<0) and (display_type!="power_fist") and (display_type!="lightning_claw") then ui_xmod[rl]=ui_xmod[rl]*-1;
-
-
-
-
+	if (argument0=2) /*and (termi_tartaros=0)*/ and (ui_xmod[rl]<0) and (display_type!="power_fist") and (display_type!="lightning_claw") then ui_xmod[rl]=ui_xmod[rl]*-1;
 
+	// TODO Why is there an exit here? Do you know why? Is the code bellow a WIP? is the exit supposed to be inside something else?
```
Comment (felipeStouvenel) [inline scripts/scr_ui_display_weapons/scr_ui_display_weapons.gml:347]: In here, why is there the exit; ?

File: scripts/scr_ui_display_weapons/scr_ui_display_weapons.gml [hunk 2]
```diff
@@ -8,372 +8,713 @@ function scr_ui_display_weapons(argument0, argument1, argument2) {
...
 	exit;
```
Comment (felipeStouvenel) [inline scripts/scr_ui_display_weapons/scr_ui_display_weapons.gml:348]: why is there an exit; here?

Comment (GalacticChimp) [reply scripts/scr_ui_display_weapons/scr_ui_display_weapons.gml:348]: not sure, need to wait for @ZAB909 input

**Comment by ZAB909** [comment]
Hey first timer, mind telling me your discord handle? Sorry if you've said who you are on discord and I've missed it. Just would like to know who im talking to! :)

**Comment by felipeStouvenel** [comment]
> Hey first timer, mind telling me your discord handle? Sorry if you've said who you are on discord and I've missed it. Just would like to know who im talking to! :)

Im DeadWay, did the other PRs before, somehow my vs code is linked to both my git accounts (thats why you see pushes as DeadWay but branch as my name

**Comment by ZAB909** [comment]
Looks fine to me, you need to sort out the conflict tho

**Comment by felipeStouvenel** [comment]
Conflict resolved, @GalacticChimp if ya can take a look at the exit mid script would be nice. Otherwise we can just delete it if it doesnt do anything important

**Comment by GalacticChimp** [comment]
> Conflict resolved, @GalacticChimp if ya can take a look at the exit mid script would be nice. Otherwise we can just delete it if it doesnt do anything important

sure, I'll take a look. Doesn't make sense have half a script doing nothing

**Comment by felipeStouvenel** [comment]
Solved merge conflict

**Review by GalacticChimp — APPROVED** [review]

**Pull request merged**